test/unittests/mongoBackend/mongoNotifyContext_test.cpp

```diff
@@ -980,3 +980,112 @@ TEST(mongoNotifyContextRequest, templateNotificationStdPayload)
 
   utExit();
 }
+
+
+
+/* ****************************************************************************
+*
+* templateNotificationOverrideStdHeader -
```

Comment:
This was taken care of in the last PR.
Before the 3-4 temporal unittests we have the following comment:

    //
    // FIXME: All following tests (templateNotification*) must be removed before
    //        merging into develop.
    //
    //